src/script/descriptor.cpp

```diff
@@ -1279,6 +1450,38 @@ std::unique_ptr<DescriptorImpl> ParseScript(uint32_t& key_exp_index, Span<const
         error = "Can only have raw() at top level";
         return nullptr;
     }
+    // Process miniscript expressions.
+    {
+        KeyParser parser(&out, nullptr);
+        auto node = miniscript::FromString(std::string(expr.begin(), expr.end()), parser);
```

Comment:
Perhaps for a follow-up PR, but maybe we want to change the `FromString` methods (in miniscript code, but also descriptors in general)` to take `std::string_view`s? That would avoid intermediary string objects like here.

Reply:
Since it's going to touch the Miniscript code again, i'd prefer to keep this as a follow-up if you don't mind.